src/providers/ldap/sdap_async.c

```diff
@@ -1470,14 +1466,10 @@ static errno_t sdap_get_generic_ext_step(struct tevent_req *req)
          "calling ldap_search_ext with [%s][%s].\n",
           state->filter ? state->filter : "no filter",
           state->search_base);
-    if (DEBUG_IS_SET(SSSDBG_TRACE_LIBS)) {
```

Comment:
I'm not sure how expensive removing this condition is. List of attributes can be quite long.

I do not say it is wrong. What do you thing @alexey-tikhonov ?

Reply:
This is a list of attrs we request. It shouldn't be _too_ long.

Reply:
I see. My mistake.